CMakeLists.txt

```diff
@@ -349,6 +349,10 @@ if (WIN32)
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /DEBUG /PDBCOMPRESS")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /STACK:1572864")
 
+  # Temporarily disable incremental link due to incremental linking CFG bug crashing crossgen. 
+  # See https://github.com/dotnet/coreclr/issues/12592
+  set(NO_INCREMENTAL_LINKER_FLAGS "/INCREMENTAL:NO")  
```

Comment:
Yup, still needed as of VS 2017  v15.4.5.

@fadimounir Seems like you were involved in some way. Do you know what "linker dev" this refers to so that we can find out the shipping vehicle for the fix?

Reply:
Validated that it works. Good timing of the release. Now we just need to find a way to share this with everyone working in the repo.

I think we can just merge with this workaround removed. CI and official build machines shouldn't be doing incremental builds and devs can just update their toolchain.

Reply:
If you want to remove this workaround, you need to add something to the build that prints build error when you are building with configuration that has the bug. The bug results in a bad crash that is hard and expensive to diagnose.

I do not see a problem with having this workaround in for a while. We do not get that much benefit from incremental linking in CoreCLR anyway. coreclr.dll - that is the largest .dll and that would benefit from the incremental linking the most - has incremental linking disabled for other reason.

Reply:
Okay, keeping the workaround then. I didn't know how much we're getting from the incremental linking since I don't normally work in this codebase.

Reply:
Could you please add a note to the comment that the bug was fixed in 15.5?